viewing_party/party.py

```diff
@@ -1,23 +1,130 @@
 # ------------- WAVE 1 --------------------
 
+from tests.test_constants import USER_DATA_2
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+    movie_dict["title"] = title
+    movie_dict["genre"] = genre
+    movie_dict ["rating"] = rating
+    
+    movie_info = [title,genre,rating]
+    if None in movie_info:
+        return None
+    else:
+        return movie_dict
+
+def add_to_watched(user_data,movie):
+    user_data = {}
+    user_data["watched"] = [movie]
+
+    return user_data
+
+def add_to_watchlist(user_data,movie):
+    user_data = {}
+    user_data["watchlist"] = [movie]
+
+    return user_data
+
+def watch_movie(user_data,title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+            return user_data
+    return user_data
+
 
 # -----------------------------------------
 # ------------- WAVE 2 --------------------
 # -----------------------------------------
 
+def get_watched_avg_rating(user_data):
+    watched_value = []
+    avg = 0
+    
+    for key in user_data["watched"]:
+        avg += 1
+        watched_value.append(key["rating"])

```

Comment:
Instead of `watched_value` being a list, appending to it, and then calling sum on it, a more straight forward way to find the sum of all ratings would be to create a variable and make it an int. 

You could create something like `ratings_sum = 0` then on line 49 you could write `ratings_sum += key["rating"]`

When you go to calculate the average on line 54, you can use the length of `user_data["watched"]` since we're iterating over movie element in the list